pkg/cloud/aws/services/ec2/securitygroups.go

```diff
@@ -304,6 +304,13 @@ func (s *Service) getSecurityGroupIngressRules(role v1alpha1.SecurityGroupRole)
 					s.scope.SecurityGroups()[v1alpha1.SecurityGroupNode].ID,
 				},
 			},
+			{
+				Description: "IP-in-IP (calico)",
+				Protocol:    v1alpha1.SecurityGroupProtocolIPinIP,
+				FromPort:    1,
```

Comment:
This should be `-1` to mean everything, if at all possible.

Reply:
Wouldn't the AWS APIs just ignore these values?

Reply:
It will, so it's a nit to stay in line with AWS docs rather than something concrete.